eng/config/PublishData.json

```diff
@@ -215,16 +215,25 @@
       "vsMajorVersion": 17,
       "insertionTitlePrefix": "[d17.12]"
     },
-    "main": {
+    "release/dev17.13": {
       "nugetKind": [
         "Shipping",
         "NonShipping"
       ],
       "vsBranch": "main",
       "vsMajorVersion": 17,
-      "insertionCreateDraftPR": false,
```

Comment:
nit: would prefer to leave this in so I don't have to think about what the default is.

Reply:
the diff is misleading, it's still in for main

Reply:
ah, think I see what you mean, we generally don't put this flag for release branches.